src/main/java/com/extendedclip/deluxemenus/config/DeluxeMenusConfig.java

```diff
@@ -763,23 +776,25 @@ private Map<Integer, TreeMap<Integer, MenuItem>> loadMenuItems(FileConfiguration
 
             // item flags
             if (c.contains(currentPath + "item_flags")) {
```

Comment:
Since this check is handled in getStringListFromConfig, we can remove it from here right?

Reply:
🤷🏼‍♂️ I'd leave it inside `loadMenuItems` too, just to have a visual separation for item flag loading logic